packages/fluentui/react-northstar/test/specs/components/Datepicker/navigateToNewDate-test.tsx

```diff
@@ -0,0 +1,32 @@
+import { navigateToNewDate } from 'src/components/Datepicker/navigateToNewDate';
+import { addDays, addMonths, addWeeks, compareDatePart } from '@fluentui/date-time-utilities';
+
+describe('navigateToNewDate', () => {
+  const referenceDate = new Date(2020, 0, 10);
```

Comment:
Is this a valid date with `0`?

Reply:
Yeah, counter-inuitevely, the months are 0-indexed.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth